doc-build/md055.md

```diff
@@ -0,0 +1,20 @@
+This rule is triggered when a table is missing a border. Borders consist of the
+pipe '|' character.
+
+This table is missing a border on the both sides:
+
+```markdown
+| Heading | Heading |
+|---------|---------
+  Data    | Data    |
+```
+
+To fix this, add a pipe character at the start or end of the row.
+
+```markdown
+| Heading | Heading |
+|---------|---------|
+| Data    | Data    |
+```
+
+Rationale: Some renderers may render tables without borders incorrectly.
```

Comment:
"some parsers may have difficult with tables that are missing their leading or trailing pipe characters"